azure-core/azure/core/pipeline/transport/aiohttp.py

```diff
@@ -49,21 +50,28 @@ class AioHttpTransport(AsyncHttpTransport):
     """AioHttp HTTP sender implementation.
     """
 
-    def __init__(self, configuration=None, *, loop=None):
+    def __init__(self, configuration=None, *, session=None, loop=None, external_owner=False):
```

Comment:
aiohttp uses `connector_owner` as the parameter name to indicate the equivalent transfer (or not) of ownership. I prefer this naming pattern since external_owner does not indicate what is owned by an external entity.

Reply:
How about `session_owner` - as it displays a more definitive correlation to the `session` that is being passed in.